.husky/pre-commit

```diff
@@ -2,7 +2,10 @@
 . "$(dirname "$0")/_/husky.sh"
 
 STAGED_FILES_TO_LINT=$(git diff --name-only --cached --diff-filter=d -- "*.ts" "*.js")
+VERSION_FILE=$(dirname "$0")/../tools/base/src/version.js
 yarn eslint -f pretty $STAGED_FILES_TO_LINT
 yarn analyze
 yarn lint:css
-yarn pretty-quick --staged
\ No newline at end of file
+yarn pretty-quick --staged
+yarn genversion --es6 --semi $VERSION_FILE
```

Comment:
@Westbrook  I have verified that upon running the `lerna publish` command locally, it automatically makes a commit. During this process, the above-mentioned workflow is triggered, which updates the version file to 0.38.0.